src/util/FactorUtil.js

```diff
@@ -347,12 +347,18 @@ function (Okta, TimeUtil) {
   };
 
   fn.removeRequirementsFromError = function (responseJSON, policy) {
-    var passwordRequirementsAsString = this.getPasswordComplexityDescription(policy);
+    var passwordRequirementsAsString = fn.getPasswordComplexityDescription(policy);
```

Comment:
This was necessary as the `this` context is _not_ the fn object because of the way the functions are imported and exported out of AuthenticatorUtil. However, I'm going to move this code to `AuthenticatorUtil` as it makes a better home to OIE code.